docs/provisioning_guide/src/asciidoc/_chapters/prepare.adoc

```diff
@@ -0,0 +1,379 @@
+* passwordless ssh to all nodes on the cluster where Trafodion will be installed.
+
+NOTE: You may need to request permission from your cluster-management team to obtain this type of access.
+
+The following example shows how to set up your user ID to have "passwordless ssh" abilities.
+
+Do the following on the Provision Master Node:
+
+```
+echo -e 'y\n' | ssh-keygen -t rsa -N "" -f $HOME/.ssh/id_rsa
+cat $HOME/.ssh/id_rsa.pub >> $HOME/.ssh/authorized_keys
+echo localhost $(cat /etc/ssh/ssh_host_rsa_key.pub) >> $HOME/.ssh/known_hosts
+echo "NoHostAuthenticationForLocalhost=yes" >> $HOME/.ssh/config
+chmod 600 $HOME/.ssh/config
+```
+
+After running these commands, do the following:
+
+* Copy the content of the `$HOME/.ssh/id_rsa.pub` file (public key) and append it to the
+`$HOME/.ssh/authorized_keys` file on the other nodes in your cluster.
+* Copy the `$HOME/.ssh/id_rsa` file (private key) from the current node to the `$HOME/.ssh` directory on the other nodes in the cluster.
+* Secure `$HOME/.ssh/id_rsa` on all nodes as private to yourself (`chmod 700`).
+* `ssh` to the other nodes in the cluster. Answer `y` to the prompt asking you whether to continue the connection.
+This adds the node to the `$HOME/.ssh/known_hosts` file completing the passwordless ssh setup.
+
+
+[[prepare-disable-requiretty]]
+== Disable requiretty
+You need to disable `requiretty` in `/etc/sudoers` on all nodes in the cluster
+to ensure that `sudo` commands can be run from inside the installation scripts.
+
+Comment out the `Defaults requiretty` setting in the `/etc/sudoers` file to
+ensure that the `requiretty` option is NOT being used.
+
+[[prepare-verify-os-requirements-and-recommendations]]
+== Verify OS Requirements and Recommendations
+
+Please ensure that the <<requirements-os-requirements-and-recommendations,OS Requirements and Recommendations>>
+are met for each node in the cluster where you intend to install Trafodion.
+
+<<<
+[[prepare-configure-ldap-identity-store]]
+== Configure LDAP Identity Store
+
+If you plan to enable security in Trafodion, then you need to have an LDAP identity store available to perform authentication.
+The Trafodion Installer prompts you to set up an authentication configuration file that points to an LDAP server (or servers),
+which enables security (that is, authentication and authorization) in the Trafodion database.
+
+If you wish to manually set up the authentication configuration file and enable security, then refer to the section on
+<<enable-security,Enable Security>>.
+
+[[prepare-gather-configuration-information]]
+== Gather Configuration Information
+
+You need to gather/decide information about your environment to aid installation Trafodion, both for the Trafodion Installer
+and for recipe-based provisioning. (In alphabetical order to make it easier to find information later on in the provisioning process.)
```

Comment:
Grammar. The parenthetical remark is not a complete sentence. Possible wordsmith: "Do so in alphabetical order..."